--- SoundStudio/build.orig	Mon Nov 20 16:37:41 2000
+++ SoundStudio/build	Tue Jul 31 08:32:25 2001
@@ -8,7 +8,7 @@
 # Set your system-dependencies here!
 
 # Which compiler would you like to invoke?
-CC="gcc -O2"
+CC="gcc ${OPT}"
 
 if [ $# = 0 -o $# -gt 1 ] ; then
     printf "%s\n" "Usage: build (config|default|install|dfltinst|backup)"
@@ -56,6 +56,7 @@
 	     fi
 	     sedCmd2="s:@TEMP_DIR@:$TEMP_DIR:"
 
+	     if [ -z "$SOX_DIR" ]; then
 	     if SOX="`which sox`" ; then
 		 SOX_DIR=`dirname $SOX`
  	         if [ "$Ask" != "Don't" ] ; then
@@ -94,6 +95,7 @@
 		     printf "\n"
      		 fi
 	     fi
+	     fi
 	     sedCmd3="s:@SOX_DIR@:$SOX_DIR:"
 
 	     printf "%s\n" "*** Building script \"studio\""
