--- SoundStudio-0.8.2/studio.orig	Tue May 19 15:54:17 1998
+++ SoundStudio-0.8.2/studio	Tue May 19 15:55:17 1998
@@ -17,13 +17,13 @@
 # Please set them as is applicable.
 
 # I need the full path Where my files are;
-set MYFILES 	.
+set MYFILES /usr/lib/SoundStudio
 
 # I need a directory to store my temporary files;
 set TEMP_DIR 	/tmp
 
 # Where do I find Sox?
-set SOX_DIR $MYFILES
+set SOX_DIR /usr/bin
 
 ##################################
 # Please Leave the Rest to me.....
